src/lsptoolshost/debugger/debugger.ts

```diff
@@ -63,8 +63,13 @@ export function registerDebugger(
     );
 }
 
-async function promptForDevKitDebugConfigurations(): Promise<boolean> {
+async function promptForDevKitDebugConfigurations(skipPrompt: boolean = false): Promise<boolean> {
```

Comment:
Update to use new object, and no need to use a default here.

```suggestion
async function promptForDevKitDebugConfigurations(options: { skipPrompt?: boolean }): Promise<boolean> {
```